css/combinedStyle.css

```diff
@@ -0,0 +1,13 @@
+/*Stying for Tables made with div tags instead of table tags
+Adapted from: 
```

Comment:
Does that mean I have to have a separate script in package.json to make it generate combinedStyle.css?